src/utils/timeformat.ts

```diff
@@ -0,0 +1,22 @@
+import dayjs from 'dayjs';
+
+export const formatTimeStr = (targetDay, platformDay, type: 'end' | 'start') => {
+  const isStart = type === 'start';
+  const targetTime = dayjs()
+    .utc()
+    .add(targetDay - platformDay - (isStart ? 0 : 1), 'day')
+    .hour(17);
+  const timeLeft = targetTime.diff(dayjs().utc(), 'hours');
+
+  if (timeLeft < 0) return '';
+  if (isStart) return targetTime.fromNow(true);
```

Comment:
Why does it return in this case? I think we also have to use the format of the end time in this case.
`Starting in 2 days, 2 hours`

Reply:
@parrot9design please confirm the format in this case.

Reply:
It should show the days and hours until voting starts/ends.